# Generated by Django 2.2.2 on 2019-07-09 19:52

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Category',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('office', models.CharField(max_length=50, verbose_name='Office ')),
                ('kitchen', models.CharField(max_length=50, verbose_name='Kitchen Ware')),
                ('entertainment', models.CharField(max_length=50, verbose_name='Entertainment')),
                ('laundry', models.CharField(max_length=50, verbose_name='Laundry')),
            ],
        ),
        migrations.CreateModel(
            name='Repairs',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('customer_name', models.CharField(max_length=120, verbose_name='Customer Name')),
                ('item_name', models.CharField(max_length=60, verbose_name='Item Name')),
                ('serial_number', models.CharField(max_length=60, verbose_name='Serial Number')),
                ('manufacturer', models.CharField(max_length=50, verbose_name='Manufacturer')),
                ('photo', models.ImageField(upload_to='uploads/')),
                ('fault_date', models.DateField(auto_now_add=True)),
                ('date_reported', models.DateTimeField(auto_now_add=True)),
                ('fault_description', models.TextField()),
            ],
        ),
    ]
